package.json

```diff
@@ -36,19 +36,21 @@
   },
```

Comment:
I think the version of this service should be bumped, not sure if it is a good idea to maintain as `"0.1.0"` with all these changes.